prime-router/metadata/file_name_templates/file-name-templates.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: standard
+  lowerCase: true
+  elements:
+    - schemaBaseName()
+    - "-"
+    - createdDate()
+    - "-"
+    - uuid()
+
+
+- name: ohio
+  upperCase: true
+  elements:
+    - cdcprime_
+    - createdDate()
+
+
+- name: aphl_light
+  lowerCase: true
+  elements:
+    - cdcprime_
+    - receivingOrganization()
+    - "_"
+    - processingModeCode()
+    - "_"
+    - createdDate()
+
+
+- name: aphl
+  lowerCase: true
+  elements:
+    - cdcprime_cdcprime_
+    - receivingOrganization()
+    - "_"
+    - processingModeCode()
+    - "_"
+    - processingModeCode()
+    - "_"
+    - createdDate()
+
```

Comment:
These are the first predefined file name formats. They're part of metadata now. *NOTE* the file extension doesn't happen here.